### PR TITLE
Update ed from 1.14.2 to 1.15

### PR DESCRIPTION
**Pull request by cstrouse**
Tested on ARM.

### CODE REVIEW

File: packages/ed.rb [hunk 1]
```diff
@@ -3,36 +3,28 @@
 class Ed < Package
   description 'GNU ed is a line-oriented text editor.'
   homepage 'http://www.gnu.org/software/ed/ed.html'
-  version '1.14.2-1'
-  source_url 'http://ftpmirror.gnu.org/ed/ed-1.14.2.tar.lz'
-  source_sha256 'f57962ba930d70d02fc71d6be5c5f2346b16992a455ab9c43be7061dec9810db'
+  version '1.15'
+  source_url 'https://ftpmirror.gnu.org/gnu/ed/ed-1.15.tar.lz'
```
Comment (uberhacker) [inline packages/ed.rb]: `https://ftpmirror.gnu.org/ed/ed-1.15.tar.lz` works also.

File: packages/ed.rb [hunk 2]
```diff
@@ -3,36 +3,28 @@
...
   })
 
   # only lz archive is available for ed and it requires lzip.
   depends_on 'lzip' => :build
 
   def self.build
-    system "./configure"
-    system "make"
+    system './configure'
+    system 'make'
```
Comment (uberhacker) [inline packages/ed.rb]: Probably should add `--prefix=#{CREW_PREFIX}` and `--libdir=#{CREW_LIB_PREFIX}` if there are library files.

File: packages/ed.rb (hunk 1 above)
Comment (cstrouse) [reply packages/ed.rb]: Done.

File: packages/ed.rb (hunk 2 above)
Comment (cstrouse) [reply packages/ed.rb]: There are no libs. I abstracted the prefix.

**Pull request merged**